optimism2/prices/insert_approx_prices_from_dex_data.sql

```diff
@@ -437,13 +438,17 @@ WHERE NOT EXISTS (SELECT * FROM prices.approx_prices_from_dex_data WHERE hour >=
 SELECT prices.insert_approx_prices_from_dex_data('2021-12-01', '2021-12-14')
 WHERE NOT EXISTS (SELECT * FROM prices.approx_prices_from_dex_data WHERE hour >= '2021-12-01' and hour < '2021-12-14');
 
+SELECT prices.insert_approx_prices_from_dex_data('2021-12-14', '2021-12-20')
+WHERE NOT EXISTS (SELECT * FROM prices.approx_prices_from_dex_data WHERE hour >= '2021-12-14' and hour < '2021-12-20');
+
 -- Have the insert script run twice every hour at minute 16 and 46
 -- `start-time` is set to go back three days in time so that entries can be retroactively updated 
 -- in case `dex.trades` or price data falls behind.
 INSERT INTO cron.job (schedule, command)
 VALUES ('16,46 * * * *', $$
     SELECT prices.insert_approx_prices_from_dex_data(
         (SELECT date_trunc('hour', now()) - interval '3 days'),
-        (SELECT date_trunc('hour', now())));
```

Comment:
Why did you make this change?

Reply:
Trying to keep the table as current as possible. I think truncating it to hours here would lead us to only get prices for 1 hour ago the latest, right?

Reply:
Makes sense. If you choose this approach, you need to be extra careful to apply `date_trunc` in the script in all the right places.

Will update the cron job now and merge this PR.